Sample Data/assessment.ttl

```diff
@@ -0,0 +1,56 @@
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix ies: <http://ies.data.gov.uk/ontology/ies4#> .
+@prefix data: <http://data.gov.uk/testdata#> .
+
+data:Scenario1 a ies:PossibleWorld .
+data:Scenario2 a ies:PossibleWorld .
+data:Scenario3 a ies:PossibleWorld .
+data:Vladimir a ies:Person .
+data:Barry a ies:Person .
+data:Fred a ies:Person .
+data:ddosAttackOnAcmeInc a ies:Hacking .
+
+data:BarHackingS3 ies:isPartOf data:Scenario3 ;
+  ies:isParticipantIn data:ddosAttackOnAcmeInc ;
+  ies:isParticipationOf data:Barry ;
+  a ies:Perpetrator .
+
+data:BarryHackingS2 ies:isPartOf data:Scenario2 ;
+  ies:isParticipantIn data:ddosAttackOnAcmeInc ;
+  ies:isParticipationOf data:Barry ;
+  a ies:Perpetrator .
+
+data:FreHackingS3 ies:isPartOf data:Scenario3 ;
+  ies:isParticipantIn data:ddosAttackOnAcmeInc ;
+  ies:isParticipationOf data:Fred ;
+  a ies:Perpetrator .
+
+data:FredHackingS1 ies:isPartOf data:Scenario1 ;
+  ies:isParticipantIn data:ddosAttackOnAcmeInc ;
+  ies:isParticipationOf data:Fred ;
+  a ies:Perpetrator .
+
+data:S1Assessment ies:assessed data:Scenario1 ;
+  ies:confidence "HIGH" ;
```

Comment:
The confidence predicates here are ```ies:hmlConfidence``` rather than the more generic ```ies:confidence``` `` predicate. Admittedly, the diagram in the document doesn't help as it does not specify them - I have made a not of this and will update

Reply:
As promised, I have now made the change to the diagram in the standard